docs/kratos/passwordless/05_passkeys.mdx

```diff
@@ -154,3 +168,159 @@ passkey found on the device will be used to sign in.
 
 </BrowserWindow>
 ```
+
+## WebAuthn
+
+:::note
+
+Please use the PassKey method instead. This is documented for legacy reasons.
```

Comment:
This isn't true, WebAuthn Passwordless is deprecated. WebAuthn is still a 2FA method for hardware keys.

Reply:
My bad, this is the passwordless section. It wasn't clear to me, in that case this is correct

Reply:
We can also explicitly mention that it is legacy only for first-factor but you can still use it for second factor 🤔

Reply:
Sure!